Bottom: 16a8e8b4f2e2982c882673d4676e30b2fda42e31
Top:    5a0a3a7c4f591820e13d6aea4f3061a96e4b0cc3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 02:36:36 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 507aa78..5a44fb8 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
